packages/Monorepo/src/Worker/RepositoryWorker.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\Monorepo\Worker;
+
+use GitWrapper\GitWorkingCopy;
+
+final class RepositoryWorker
+{
+    public function mergeRepositoryToMonorepoDirectory(string $gitRepository, GitWorkingCopy $gitWorkingCopy): void
+    {
+        $remoteName = md5($gitRepository);
```

Comment:
why?

Reply:
Ok, it's still needed. So I've created a special method for that with clear name